.github/workflows/build.yml

```diff
@@ -0,0 +1,41 @@
+name: Actions 😎
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Automating Map Editor Build
+    runs-on: windows-latest
```

Comment:
Please add a comment why we use Windows here, as Ubuntu is standard (and also cheaper if costs were relevant)